src/vs/editor/common/modes/languageConfiguration.ts

```diff
@@ -61,7 +61,7 @@ export interface LanguageConfiguration {
 	 * selected string is surrounded by the open and close characters. If not set, the autoclosing pairs
 	 * settings will be used.
 	 */
-	surroundingPairs?: IAutoClosingPair[];
+	surroundingPairs?: IAutoClosingPairConditional[];
```

Comment:
In the original repo, this type was not changed - https://github.com/microsoft/vscode/blob/5a139d4ffc72fb8100c789bf2f67c09e5c33d05c/src/vs/editor/common/languages/languageConfiguration.ts#L64
Did you open any discussion or issue in the vscode main repo to discuss this change?